src/applications/hca/enrollment-status-helpers.jsx

```diff
@@ -819,8 +819,6 @@ export function getAlertType(enrollmentStatus) {
       status = DASHBOARD_ALERT_TYPES.enrolled;
       break;
 
-    case HCA_ENROLLMENT_STATUSES.activeDutyHasApplied:
-    case HCA_ENROLLMENT_STATUSES.activeDutyHasNotApplied:
```

Comment:
Removing all of the `activeDutyHasNotApplied` cases for the Dashboard alert helpers since we are no longer showing HCA-related updates on the Dashboard for this particular case.